test/jdk/java/awt/TitledBorder/TitledBorderTest.java

```diff
@@ -89,67 +87,66 @@ public static void main(String[] args) throws Exception {
     }
 
     private static void testScaling(boolean show) throws Exception {
-        SwingUtilities.invokeAndWait(() -> createAndShowGUI(show));
+        SwingUtilities.invokeAndWait(() -> createGUI(show));
 
         for (int i = 0; i < images.size(); i++) {
-                BufferedImage img = images.get(i);
-                double scaling = scales[i];
+            BufferedImage img = images.get(i);
+            double scaling = scales[i];
 
-                // For vertical count the number of shadow / highlight in
-                // the middle of the image at (x = SIZE.width / 2)
-                // (there must be no background color between these two colours)
-                // Then skip background until the next border where you count
-                // shadow and highlight thickness
+            // checking vertical border
+            int x = SIZE.width / 2;
+            checkVerticalBorder(x, img, scaling);
 
-                 int x = SIZE.width / 2;
-                 checkVerticalBorder(x, img, scaling);
+            for (Point p : panelLocations) {
+                int y = (int) (p.y * scaling) + SIZE.height / 2;
+                System.out.println(scaling + " : " + y);
 
-                // For horizontal border, take the middle of each panel and
-                // count the number of shadow and highlight pixels
-                for (Point p : panelLocations) {
-                    int y = (int) (p.y * scaling) + SIZE.height / 2;
-                    System.out.println(scaling + " : " + y);
-
-                     checkHorizontalBorder(y, img, scaling);
-                }
+                checkHorizontalBorder(y, img, scaling);
+            }
         }
     }
 
-    private static void hBorderLoop(int x1, int x2, int y, BufferedImage buff,
-                                    Color c, double scaling) throws RuntimeException {
+    private static void checkHorizontalBorder(int y, BufferedImage img, double scaling) throws RuntimeException {
```

Comment:
There's no need to declare `RuntimeException` in throws clause. If you prefer to, please wrap the line, it's longer than 100 columns.